lib/Demangling/NodePrinter.cpp

```diff
@@ -1349,6 +1349,17 @@ static bool needSpaceBeforeType(NodePointer Type) {
   }
 }
 
+static bool shouldShowEntityType(Node::Kind EntityKind,
```

Comment:
Can you add a comment with the rationale (i.e., that signatures aren't needed to uniquely identify a closure?